packages/suite-desktop-ui/src/support/DesktopUpdater/Downloading.tsx

```diff
@@ -33,8 +33,8 @@ const Text = styled(H2)`
     font-weight: ${variables.FONT_WEIGHT.MEDIUM};
 `;
 
-const StyledButton = styled(Button)`
-    margin: 21px 0;
+const StyledProgress = styled(Progress)`
+    margin-top: 16px;
```

Comment:
```suggestion
    margin-top: ${spacingsPx.md};
```

Reply:
later, don't want to add a new dependancy in this pr